### ISSUE TITLE
pan.textSize does not take trailing whitespace into account

### ISSUE THREAD

**Issue by liquidev** [liquidev/pan#10]
This is a bit of a problem with cairo's text extents, but it's possible to circumvent this:

```lua
local spacew = pan.textSize(font, " w", size)
local w = pan.textSize(font, "w", size)
local space = spacew - w
```
and then writing a wrapper over textSize that returns the width with trailing whitespace added.

**Comment by liquidev** [comment]
Fixed in `68f779f`.

**Comment by liquidev** [comment]
Just as a note: this method is imprecise in a world where kerning exists.
